TemplateBundle/content/Application/Application.Core.Tests/Sample.fs

```diff
@@ -0,0 +1,37 @@
+namespace Application.Core.Tests
+
+module Sample =
+    open Expecto
+    open Application.Core.Say
+
+    [<Tests>]
+    let tests =
```

Comment:
If a test project is included I think we should actually test the basic app. MVU apps are great to test.

Reply:
Ah I included the tests since I feel this is like the QuickStart it should have some tests for your core lib

I can add tests to the other projects as well but I feel their use is directed to exploration/experimentation
thoughts?

Reply:
Yeah, I also think it's not required for the other templates